Update install-redis-on-linux.md (#2640)
redis/redis-doc: docs/install/install-redis/install-redis-on-linux.md

   {{< highlight bash  >}}
   redis-cli ping
   {{< / highlight  >}}   

>    If the connection is succesfful you get `PONG` in response.

Reviewer: I hope you don't mind a bit of copy-editing.

```suggestion
   If the connection is successful, `PONG` is returned.
```
